### ISSUE TITLE
Is this library no longer maintained ?

### ISSUE THREAD

**Issue by ahmedosama5200** [sohobloo/react-native-modal-dropdown#256]

**Comment by donni106** [comment]
Correct. Please have a look at this issue: https://github.com/sohobloo/react-native-modal-dropdown/issues/251
There is https://github.com/siemiatj/react-native-modal-dropdown, which gets updated.

**Comment by ahmedosama5200** [comment]
Thanks

**Comment by siemiatj** [comment]
I just got a green light from npm to take over this package and will release it later today.

**Comment by siemiatj** [comment]
Published the updated version to npm.
